module/Finna/src/Finna/RecordDriver/SolrLido.php

```diff
@@ -188,16 +241,25 @@ public function getAccessRestrictionsType($language)
      */
     public function getAllImages($language = 'fi')
     {
+        if (null !== $this->cachedImages) {
+            return $this->cachedImages;
+        }
+
         $result = [];
+        $highResolution = [];
```

Comment:
Voisiko high resolution -tiedot upottaa tuohon normaaliin cachedImages:iin? Ei ole hyvä, että getAllImages sivuvaikutuksena täyttää $this->cachedHighResolution:n ja olisi myös kiva pitää tiedot yhdessä.

Reply:
Tallennetaan nykyään omana osiona cachedImagesiin.